### PR TITLE
docs: Update test cases

### PR DESCRIPTION
**Pull request by dcalhoun**
Update test cases to note known issues and remove automated or low-value test cases. 

- docs: Capture Attachment Page known issue
- docs: Remove TC010 Columns front-end presentation test
- docs: Remove TC010 Gallery rearrange items test
- docs: Remove TC012 Gallery Link to test
- docs: Remove TC002 VideoPress caption test
- docs: Remove TC001 Color settings segmented controls test
- docs: Remove TC004 Color settings swatch press test

### CODE REVIEW

**Comment by dcalhoun** [comment]
@fluiddot removing the color settings animation test cases is aggressive. Feel free to push back if you disagree.

File: test-cases/gutenberg/color-settings.md
Comment (dcalhoun) [inline test-cases/gutenberg/color-settings.md:1]: Testing animations — whether manually or automatically — seems like a low return of value for the effort required.

File: test-cases/gutenberg/columns.md [hunk 1]
```diff
@@ -1,19 +0,0 @@
-# Columns Block - Test Cases
-
---------------------------------------------------------------------------------
-
-##### TC010
-
-### Columns percentage mechanism - custom percentage values are rendered correctly on frontend
```
Comment (dcalhoun) [inline test-cases/gutenberg/columns.md:7]: Other automated Columns tests assert the expected mark up. I think it is safe to presume that if the mark up is correct, the front-end website will render correctly in the browser.

File: test-cases/gutenberg/gallery.md
Comment (dcalhoun) [inline test-cases/gutenberg/gallery.md:1]: These test cases were automated in https://github.com/WordPress/gutenberg/pull/38571.

File: test-cases/jetpack/videopress.md [hunk 2]
```diff
@@ -28,21 +28,6 @@ Use Simple WordPress.com site with a Premium plan. Free or Personal plans only a
 
 <img src="../../test-cases/resources/../resources/videopress-upload-video-4.png" width="360">
 
-##### TC002
```
Comment (dcalhoun) [inline test-cases/jetpack/videopress.md:31]: This test case was automated in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5826.

File: test-cases/gutenberg/color-settings.md
Comment (fluiddot) [reply test-cases/gutenberg/color-settings.md:1]: I completely agree, I don't see them as a critical flow that needs to be tested.

File: test-cases/gutenberg/columns.md (hunk 1 above)
Comment (fluiddot) [reply test-cases/gutenberg/columns.md:7]: Yep, as pointed out, we are already checking the HTML content related to the column's width ([reference](https://github.com/WordPress/gutenberg/blob/58f5e9a6a972d16aeca13693ce863bcedaf342b0/packages/block-library/src/columns/test/edit.native.js#L338-L431)). So this test case is not really adding value to the test coverage 👍 .

File: test-cases/gutenberg/gallery.md
Comment (fluiddot) [reply test-cases/gutenberg/gallery.md:1]: Good catch!

File: test-cases/gutenberg/gallery.md [hunk 3]
```diff
@@ -210,45 +191,6 @@ Gallery block should allow uploading images from the iOS Files app.
...
-Gallery images should respect the `Link to` setting
-
-**Steps:**
-
-* Add a gallery block and tap "Add Media"
-* Add some images from the media library
-* Tap the block settings icon ⚙️ :gear:
-* Tap `Link to` setting
-* For each of the settings
-  * <img src="../../test-cases/resources/gallery-settings-link-to.png" width="360">
```
Comment (fluiddot) [inline test-cases/gutenberg/gallery.md:228]: This image is only referenced here so we can also remove it.

**Review by fluiddot — APPROVED** [review]
LGTM 🎊 ! Thanks @dcalhoun for cleaning up the test cases 🙇 !

I only added a comment regarding an image we could also remove as won't be referenced anymore. Apart from that, the PR is good to go 🎊 .

**Pull request merged**